### ISSUE TITLE
Multilingual transformer: load_state_dict() got an unexpected keyword argument 'args'

### ISSUE THREAD

**Issue by Sun694** [pytorch/fairseq#2088]
## 🐛 Bug

Upon loading statedict, I get the error "load_state_dict() got an unexpected keyword argument 'args'".

If I stop training, then try to reload a checkpoint, it throws the error: "Exception: Cannot load model parameters from checkpoint [PATH]; please ensure that the architectures match.". 

### To Reproduce

Steps to reproduce the behavior (**always include the command you ran**):

I wanted to train a model that could do A->B and B->A, so I used the multilingual_transformer arch and multilingual_translation task.

#### Code sample
```python
! fairseq-preprocess \
    -s s -t d\
    --joined-dictionary \
    --trainpref PATH \
    --validpref PATH \
    --testpref PATH \
    --destdir PATH \
    --workers 20 \
    --padding-factor 1

delete dictionary made in previous step

! fairseq-preprocess \
    -s d -t s\
    --joined-dictionary \
    --trainpref PATH \
    --validpref PATH \
    --testpref PATH \
    --destdir PATH \
    --workers 20 \
    --padding-factor 1

! fairseq-train --task multilingual_translation \
  PATH \
  --save-dir PATH  \
  --arch multilingual_transformer \
  --lang-pairs s-d,d-s \
  --dropout 0.1 \
  --save-interval 1\
  --optimizer adam --adam-betas '(0.9, 0.98)' --weight-decay 0.01 --clip-norm 0.0 \
  --reset-optimizer \
  --lr 0.0005 --lr-scheduler inverse_sqrt --warmup-updates 4000 --warmup-init-lr 1e-07 \
  --max-tokens 2048 --update-freq 8 \
  --max-update 50000 \
  --max-epoch 1 \
```

Then, after 1 epoch, try to restart training, or use fairseq-generate or fairseq-interactive.

### Expected behavior

I expect to be able to load the model.

### Environment

 - fairseq Version (e.g., 1.0 or master): master
 - PyTorch Version (e.g., 1.0) 1.5
 - OS (e.g., Linux): Linux
 - How you installed fairseq (`pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.6
 - CUDA/cuDNN version: 10.1
 - GPU models and configuration: P100

### Additional context

I found error: https://github.com/pytorch/fairseq/issues/1393

I even verified the multilingual transformer on my local fairseq had the `args=None` as a parameter for the `load_state_dict() function`. However, I believe some implicit loading function picked the incorrect model architecture, and I cannot find a way to force it to use the correct one.

Here is the entire traceback for fairseq generate and fairseq train (in order):

```python
Namespace(beam=5, bpe=None, cpu=False, criterion='cross_entropy', data='/content/drive/My Drive/biochem-rep-learning/fairseq_process', dataset_impl=None, decoder_langtok=False, decoding_format=None, diverse_beam_groups=-1, diverse_beam_strength=0.5, empty_cache_freq=0, encoder_langtok=None, force_anneal=None, fp16=False, fp16_init_scale=128, fp16_scale_tolerance=0.0, fp16_scale_window=None, gen_subset='test', iter_decode_eos_penalty=0.0, iter_decode_force_max_iter=False, iter_decode_max_iter=10, lang_pairs='s-d,d-s', lazy_load=False, left_pad_source='True', left_pad_target='False', lenpen=1, log_format=None, log_interval=1000, lr_scheduler='fixed', lr_shrink=0.1, match_source_len=False, max_len_a=0, max_len_b=200, max_sentences=None, max_source_positions=1024, max_target_positions=1024, max_tokens=12000, memory_efficient_fp16=False, min_len=1, min_loss_scale=0.0001, model_overrides='{}', momentum=0.99, nbest=1, no_beamable_mm=False, no_early_stop=False, no_progress_bar=False, no_repeat_ngram_size=0, num_shards=1, num_workers=1, optimizer='nag', path='/content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_best.pt', prefix_size=0, print_alignment=False, print_step=False, quiet=False, raw_text=False, remove_bpe=None, replace_unk=None, required_batch_size_multiple=8, results_path=None, retain_iter_history=False, sacrebleu=False, sampling=False, sampling_topk=-1, sampling_topp=-1.0, score_reference=False, seed=1, shard_id=0, skip_invalid_size_inputs_valid_test=False, source_lang='d', target_lang='s', task='multilingual_translation', temperature=1.0, tensorboard_logdir='', threshold_loss_scale=None, tokenizer=None, unkpen=0, unnormalized=False, upsample_primary=1, user_dir=None, warmup_updates=0, weight_decay=0.0)
| [d] dictionary: 355 types
| [s] dictionary: 355 types
| loaded 999 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/test.d-s.d
| loaded 999 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/test.d-s.s
| /content/drive/My Drive/biochem-rep-learning/fairseq_process test d-s 999 examples
| loading model(s) from /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_best.pt
/usr/local/lib/python3.6/dist-packages/fairseq/models/fairseq_model.py:280: UserWarning: FairseqModel is deprecated, please use FairseqEncoderDecoderModel or BaseFairseqModel instead
  for key in self.keys
Traceback (most recent call last):
  File "/usr/local/bin/fairseq-generate", line 8, in <module>
    sys.exit(cli_main())
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/generate.py", line 199, in cli_main
    main(args)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/generate.py", line 47, in main
    task=task,
  File "/usr/local/lib/python3.6/dist-packages/fairseq/checkpoint_utils.py", line 179, in load_model_ensemble
    ensemble, args, _task = load_model_ensemble_and_task(filenames, arg_overrides, task)
  File "/usr/local/lib/python3.6/dist-packages/fairseq/checkpoint_utils.py", line 198, in load_model_ensemble_and_task
    model.load_state_dict(state["model"], strict=True, args=args)
TypeError: load_state_dict() got an unexpected keyword argument 'args'


FAIRSEQ TRAIN ERROR


Namespace(activation_dropout=0.0, activation_fn='relu', adam_betas='(0.9, 0.98)', adam_eps=1e-08, adaptive_input=False, adaptive_softmax_cutoff=None, adaptive_softmax_dropout=0, arch='multilingual_transformer', attention_dropout=0.0, best_checkpoint_metric='loss', bpe=None, bucket_cap_mb=25, clip_norm=0.0, cpu=False, criterion='cross_entropy', cross_self_attention=False, curriculum=0, data='/content/drive/My Drive/biochem-rep-learning/fairseq_process', dataset_impl=None, ddp_backend='c10d', decoder_attention_heads=8, decoder_embed_dim=512, decoder_embed_path=None, decoder_ffn_embed_dim=2048, decoder_input_dim=512, decoder_langtok=False, decoder_layerdrop=0, decoder_layers=6, decoder_layers_to_keep=None, decoder_learned_pos=False, decoder_normalize_before=False, decoder_output_dim=512, device_id=0, disable_validation=False, distributed_backend='nccl', distributed_init_method=None, distributed_no_spawn=False, distributed_port=-1, distributed_rank=0, distributed_world_size=1, dropout=0.1, empty_cache_freq=0, encoder_attention_heads=8, encoder_embed_dim=512, encoder_embed_path=None, encoder_ffn_embed_dim=2048, encoder_langtok=None, encoder_layerdrop=0, encoder_layers=6, encoder_layers_to_keep=None, encoder_learned_pos=False, encoder_normalize_before=False, fast_stat_sync=False, find_unused_parameters=False, fix_batches_to_gpus=False, fixed_validation_seed=None, fp16=False, fp16_init_scale=128, fp16_scale_tolerance=0.0, fp16_scale_window=None, keep_interval_updates=-1, keep_last_epochs=-1, lang_pairs='s-d,d-s', layer_wise_attention=False, layernorm_embedding=False, lazy_load=False, left_pad_source='True', left_pad_target='False', log_format=None, log_interval=1000, lr=[0.0005], lr_scheduler='inverse_sqrt', max_epoch=1, max_sentences=None, max_sentences_valid=None, max_source_positions=1024, max_target_positions=1024, max_tokens=2048, max_tokens_valid=2048, max_update=50000, maximize_best_checkpoint_metric=False, memory_efficient_fp16=False, min_loss_scale=0.0001, min_lr=-1, no_cross_attention=False, no_epoch_checkpoints=False, no_last_checkpoints=False, no_progress_bar=False, no_save=False, no_save_optimizer_state=False, no_scale_embedding=False, no_token_positional_embeddings=False, num_workers=1, optimizer='adam', optimizer_overrides='{}', raw_text=False, required_batch_size_multiple=8, reset_dataloader=False, reset_lr_scheduler=False, reset_meters=False, reset_optimizer=True, restore_file='checkpoint_last.pt', save_dir='/content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq', save_interval=1, save_interval_updates=0, seed=1, sentence_avg=False, share_all_embeddings=False, share_decoder_embeddings=False, share_decoder_input_output_embed=False, share_decoders=False, share_encoder_embeddings=False, share_encoders=False, skip_invalid_size_inputs_valid_test=False, source_lang=None, target_lang=None, task='multilingual_translation', tensorboard_logdir='', threshold_loss_scale=None, tokenizer=None, train_subset='train', update_freq=[8], upsample_primary=1, use_bmuf=False, user_dir=None, valid_subset='valid', validate_interval=1, warmup_init_lr=1e-07, warmup_updates=4000, weight_decay=0.01)
| [d] dictionary: 355 types
| [s] dictionary: 355 types
| loaded 499 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/valid.s-d.s
| loaded 499 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/valid.s-d.d
| /content/drive/My Drive/biochem-rep-learning/fairseq_process valid s-d 499 examples
| loaded 499 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/valid.d-s.d
| loaded 499 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/valid.d-s.s
| /content/drive/My Drive/biochem-rep-learning/fairseq_process valid d-s 499 examples
/usr/local/lib/python3.6/dist-packages/fairseq/models/fairseq_model.py:280: UserWarning: FairseqModel is deprecated, please use FairseqEncoderDecoderModel or BaseFairseqModel instead
  for key in self.keys
MultilingualTransformerModel(
--snip architecture print statement--
| model multilingual_transformer, criterion CrossEntropyCriterion
| num. model params: 89367552 (num. trained: 89367552)
| training on 1 GPUs
| max tokens per GPU = 2048 and max sentences per GPU = None
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/fairseq/trainer.py", line 195, in load_checkpoint
    state["model"], strict=True, args=self.args
TypeError: load_state_dict() got an unexpected keyword argument 'args'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/fairseq-train", line 8, in <module>
    sys.exit(cli_main())
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 333, in cli_main
    main(args)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 70, in main
    extra_state, epoch_itr = checkpoint_utils.load_checkpoint(args, trainer)
  File "/usr/local/lib/python3.6/dist-packages/fairseq/checkpoint_utils.py", line 120, in load_checkpoint
    reset_meters=args.reset_meters,
  File "/usr/local/lib/python3.6/dist-packages/fairseq/trainer.py", line 204, in load_checkpoint
    "please ensure that the architectures match.".format(filename)
Exception: Cannot load model parameters from checkpoint /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt; please ensure that the architectures match.```

**Comment by AmrHendy** [comment]
You will need to change line 195 in "**/usr/local/lib/python3.6/dist-packages/fairseq/trainer.py**" with **strict=False** rather than strict=True to make loading the checkpoints working at the current time.

**Comment by myleott** [comment]
I'm not able to reproduce this on master. Are you using the fairseq 0.9 or master? Can you try again on the latest master?

**Comment by Sun694** [comment]
> I'm not able to reproduce this on master. Are you using the fairseq 0.9 or master? Can you try again on the latest master?

I was mistaken. I was not on master, I was on .9.

I tried again on master. Now, when training, saving fails. The command is the same as my first post, the new traceback is:

```py
2020-05-06 00:35:45 | INFO | fairseq_cli.train | model multilingual_transformer, criterion CrossEntropyCriterion
2020-05-06 00:35:45 | INFO | fairseq_cli.train | num. model params: 89508864 (num. trained: 89508864)
2020-05-06 00:35:48 | INFO | fairseq_cli.train | training on 1 GPUs
2020-05-06 00:35:48 | INFO | fairseq_cli.train | max tokens per GPU = 2048 and max sentences per GPU = None
2020-05-06 00:35:48 | INFO | fairseq.trainer | no existing checkpoint found /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt
2020-05-06 00:35:48 | INFO | fairseq.trainer | loading train data for epoch 1
2020-05-06 00:35:48 | INFO | fairseq.data.data_utils | loaded 65000 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.s-d.s
2020-05-06 00:35:48 | INFO | fairseq.data.data_utils | loaded 65000 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.s-d.d
2020-05-06 00:35:48 | INFO | fairseq.tasks.translation | /content/drive/My Drive/biochem-rep-learning/fairseq_process train s-d 65000 examples
2020-05-06 00:35:48 | INFO | fairseq.data.data_utils | loaded 65000 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.d-s.d
2020-05-06 00:35:48 | INFO | fairseq.data.data_utils | loaded 65000 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.d-s.s
2020-05-06 00:35:48 | INFO | fairseq.tasks.translation | /content/drive/My Drive/biochem-rep-learning/fairseq_process train d-s 65000 examples
epoch 001:  99% 182/183 [04:58<00:01,  1.66s/it, loss=6.297, sample_size=25779.4, nsentences=708.94, ntokens=25779.4, ppl=78.62, wps=15766.4, ups=0.61, wpb=25779.4, bsz=708.9, num_updates=100, lr=1.25975e-05, gnorm=4.686, train_wall=156, wall=165]Traceback (most recent call last):
  File "/usr/local/bin/fairseq-train", line 8, in <module>
    sys.exit(cli_main())
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 359, in cli_main
    main(args)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 117, in main
    valid_losses = train(args, trainer, task, epoch_itr, max_update)
  File "/usr/lib/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 201, in train
    valid_losses = validate_and_save(args, trainer, task, epoch_itr, valid_subsets)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 241, in validate_and_save
    valid_losses = validate(args, trainer, task, epoch_itr, valid_subsets)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 278, in validate
    num_workers=args.num_workers,
  File "/usr/local/lib/python3.6/dist-packages/fairseq/tasks/fairseq_task.py", line 164, in get_batch_iterator
    indices = dataset.ordered_indices()
  File "/usr/local/lib/python3.6/dist-packages/fairseq/data/round_robin_zip_datasets.py", line 97, in ordered_indices
    for key, dataset in self.datasets.items()
  File "/usr/local/lib/python3.6/dist-packages/fairseq/data/round_robin_zip_datasets.py", line 97, in <listcomp>
    for key, dataset in self.datasets.items()
  File "/usr/local/lib/python3.6/dist-packages/fairseq/data/language_pair_dataset.py", line 279, in ordered_indices
    indices = indices[np.argsort(self.tgt_sizes[indices], kind='mergesort')]
IndexError: index 5361 is out of bounds for axis 0 with size 4999```

**Comment by Sun694** [comment]
> That seems to suggest that the target side of the data has a different number of examples as the source side.
> 
> Can you share the log from your fairseq-preprocess command?

You are correct. They were different sizes. I had manually typed the partitions and made an error. I fixed this. I then attempted training again, and got no error when saving. Below is the output of the end of training. This training command is the exact same as the OP.


`| model multilingual_transformer, criterion CrossEntropyCriterion
| num. model params: 89367552 (num. trained: 89367552)
| training on 1 GPUs
| max tokens per GPU = 2048 and max sentences per GPU = None
| no existing checkpoint found /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt
| loading train data for epoch 0
| loaded 6500 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.s-d.s
| loaded 6500 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.s-d.d
| /content/drive/My Drive/biochem-rep-learning/fairseq_process train s-d 6500 examples
| loaded 6500 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.d-s.d
| loaded 6500 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/train.d-s.s
| /content/drive/My Drive/biochem-rep-learning/fairseq_process train d-s 6500 examples
| epoch 001:   0% 0/20 [00:00<?, ?it/s]/pytorch/torch/csrc/utils/python_arg_parser.cpp:756: UserWarning: This overload of add_ is deprecated:
	add_(Number alpha, Tensor other)
Consider using one of the following signatures instead:
	add_(Tensor other, *, Number alpha)
| epoch 001 | loss 17.101 | ppl 140574 | wps 13999 | ups 1 | wpb 24974.100 | bsz 650.000 | num_updates 20 | lr 2.5995e-06 | gnorm 11.567 | clip 0.000 | oom 0.000 | wall 36 | train_wall 33 | s-d:loss 8.24907 | s-d:ntokens 13000 | s-d:nsentences 325 | s-d:sample_size 13000 | d-s:loss 8.8519 | d-s:ntokens 11974.1 | d-s:nsentences 325 | d-s:sample_size 11974.1
| epoch 001 | valid on 'valid' subset | loss 14.612 | ppl 25033.5 | num_updates 20 | s-d:loss 6.94703 | s-d:ntokens 1663.33 | s-d:nsentences 41.5833 | s-d:sample_size 1663.33 | d-s:loss 7.70874 | d-s:ntokens 1501.08 | d-s:nsentences 41.5833 | d-s:sample_size 1501.08
| saved checkpoint /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint1.pt (epoch 1 @ 20 updates) (writing took 55.43511724472046 seconds)
| done training in 92.4 seconds`

Below is the output from using fairseq-generate:

`!fairseq-generate /content/drive/My\ Drive/biochem-rep-learning/fairseq_process \
    --task multilingual_translation\
    --lang-pairs s-d,d-s \
    --path /content/drive/My\ Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt \

Namespace(beam=5, bpe=None, cpu=False, criterion='cross_entropy', data='/content/drive/My Drive/biochem-rep-learning/fairseq_process', dataset_impl=None, decoder_langtok=False, decoding_format=None, diverse_beam_groups=-1, diverse_beam_strength=0.5, empty_cache_freq=0, encoder_langtok=None, force_anneal=None, fp16=False, fp16_init_scale=128, fp16_scale_tolerance=0.0, fp16_scale_window=None, gen_subset='test', iter_decode_eos_penalty=0.0, iter_decode_force_max_iter=False, iter_decode_max_iter=10, lang_pairs='s-d,d-s', lazy_load=False, left_pad_source='True', left_pad_target='False', lenpen=1, log_format=None, log_interval=1000, lr_scheduler='fixed', lr_shrink=0.1, match_source_len=False, max_len_a=0, max_len_b=200, max_sentences=None, max_source_positions=1024, max_target_positions=1024, max_tokens=12000, memory_efficient_fp16=False, min_len=1, min_loss_scale=0.0001, model_overrides='{}', momentum=0.99, nbest=1, no_beamable_mm=False, no_early_stop=False, no_progress_bar=False, no_repeat_ngram_size=0, num_shards=1, num_workers=1, optimizer='nag', path='/content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt', prefix_size=0, print_alignment=False, print_step=False, quiet=False, raw_text=False, remove_bpe=None, replace_unk=None, required_batch_size_multiple=8, results_path=None, retain_iter_history=False, sacrebleu=False, sampling=False, sampling_topk=-1, sampling_topp=-1.0, score_reference=False, seed=1, shard_id=0, skip_invalid_size_inputs_valid_test=False, source_lang=None, target_lang=None, task='multilingual_translation', temperature=1.0, tensorboard_logdir='', threshold_loss_scale=None, tokenizer=None, unkpen=0, unnormalized=False, upsample_primary=1, user_dir=None, warmup_updates=0, weight_decay=0.0)
| [d] dictionary: 355 types
| [s] dictionary: 355 types
| loaded 999 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/test.s-d.s
| loaded 999 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/test.s-d.d
| /content/drive/My Drive/biochem-rep-learning/fairseq_process test s-d 999 examples
| loaded 999 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/test.d-s.d
| loaded 999 examples from: /content/drive/My Drive/biochem-rep-learning/fairseq_process/test.d-s.s
| /content/drive/My Drive/biochem-rep-learning/fairseq_process test d-s 999 examples
| loading model(s) from /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt
/usr/local/lib/python3.6/dist-packages/fairseq/models/fairseq_model.py:280: UserWarning: FairseqModel is deprecated, please use FairseqEncoderDecoderModel or BaseFairseqModel instead
  for key in self.keys
Traceback (most recent call last):
  File "/usr/local/bin/fairseq-generate", line 8, in <module>
    sys.exit(cli_main())
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/generate.py", line 199, in cli_main
    main(args)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/generate.py", line 47, in main
    task=task,
  File "/usr/local/lib/python3.6/dist-packages/fairseq/checkpoint_utils.py", line 179, in load_model_ensemble
    ensemble, args, _task = load_model_ensemble_and_task(filenames, arg_overrides, task)
  File "/usr/local/lib/python3.6/dist-packages/fairseq/checkpoint_utils.py", line 198, in load_model_ensemble_and_task
    model.load_state_dict(state["model"], strict=True, args=args)
TypeError: load_state_dict() got an unexpected keyword argument 'args'`

Below is the error from re-running the exact same train command as OP, after training for one epoch.

`| model multilingual_transformer, criterion CrossEntropyCriterion
| num. model params: 89367552 (num. trained: 89367552)
| training on 1 GPUs
| max tokens per GPU = 2048 and max sentences per GPU = None
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/fairseq/trainer.py", line 195, in load_checkpoint
    state["model"], strict=True, args=self.args
TypeError: load_state_dict() got an unexpected keyword argument 'args'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/fairseq-train", line 8, in <module>
    sys.exit(cli_main())
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 333, in cli_main
    main(args)
  File "/usr/local/lib/python3.6/dist-packages/fairseq_cli/train.py", line 70, in main
    extra_state, epoch_itr = checkpoint_utils.load_checkpoint(args, trainer)
  File "/usr/local/lib/python3.6/dist-packages/fairseq/checkpoint_utils.py", line 120, in load_checkpoint
    reset_meters=args.reset_meters,
  File "/usr/local/lib/python3.6/dist-packages/fairseq/trainer.py", line 204, in load_checkpoint
    "please ensure that the architectures match.".format(filename)
Exception: Cannot load model parameters from checkpoint /content/drive/My Drive/biochem-rep-learning/chckpoints/fairseq/checkpoint_last.pt; please ensure that the architectures match.`

I am able to upload my data and my full script. Would it be worthwhile to create a minimalist set of commands to generate this error?

Anything to make it easier for you. Thanks so far.
